tests/unit/amo/sagas/testLanding.js

```diff
@@ -54,17 +58,97 @@ describe(__filename, () => {
       );
     }
 
-    it('fetches landing page addons from the API', async () => {
+    it('fetches extensions landing page addons from the API', async () => {
```

Comment:
I'm not entirely sure, but I wonder if you could use `it.each()` to avoid writing two tests that are essentially the same, but just have a different input and a different expectation?